dapps/admin/src/utils/formatDate.js

```diff
@@ -1,5 +1,5 @@
 import format from 'date-fns/format'
 
 export default function formatDate(timestamp) {
-  return format(new Date(timestamp * 1000), 'MM/DD/YYYY h:mm a')
+  return format(new Date(timestamp * 1000), 'MM/dd/yyyy h:mm a')
```

Comment:
Just wondering, why not use something like `dayjs.unix(timestamp).format('MM/DD/YYYY H:MM A')`?

Reply:
don't currently have dayjs in the admin dapp. I guess we can switch it for consistency

Reply:
Got it. 👍